### PR TITLE
Added hreflang alternate tags

### LINKED ISSUE #61: Add alternate/hreflang link tags for translations
Per https://support.google.com/webmasters/answer/189077 we should make alternate translations discoverable with `<link rel="alternate" hreflang="..." href="...">` elements.

These should be available on every page for all supported translations.

This is basically the metadata/SEO equivalent of #50, so there may be some opportunities to reuse code. cc @tanhengyeow

### PR DESCRIPTION
**Pull request by AymenLoukil**
I checked out your branch "lang". Please tell me if i should wait your PR merge :) 

I loop over the available languages and add the link alternate tags to the head (on base.html). Sounds working 

Fixes #61

### CODE REVIEW

File: src/templates/base.html [hunk 1]
```diff
@@ -25,6 +25,10 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800">
     <link rel="stylesheet" href="/static/css/styles.css">
     {% endblock %}
+		{% set view_args = request.view_args.copy() %}
+	{% for l in supported_languages %}
```
Comment (rviscomi) [inline src/templates/base.html]: This includes the current language. Is that ok for alternate hreflangs?

File: src/templates/base.html [hunk 2]
```diff
@@ -25,6 +25,10 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800">
     <link rel="stylesheet" href="/static/css/styles.css">
     {% endblock %}
+		{% set view_args = request.view_args.copy() %}
```
Comment (rviscomi) [inline src/templates/base.html]: The indentation here is inconsistent. Could you match the rest of the file?

File: src/templates/base.html (hunk 1 above)
Comment (AymenLoukil) [reply src/templates/base.html]: Yes. The rules are : 
- You self reference (so if we are in /en/ we should point to /en/
- You reference tha alternate pages (so here we point to /ja/
- Bidirectional (A point to B, B should point to A)

File: src/templates/base.html (hunk 2 above)
Comment (AymenLoukil) [reply src/templates/base.html]: OK i'll correct.

Comment (rviscomi) [reply src/templates/base.html]: Looks like they're one nested level too many. Everything should be aligned with the blocks/links before it.

Comment (AymenLoukil) [reply src/templates/base.html]: Already corrected in the new commit no ?

Comment (rviscomi) [reply src/templates/base.html]: See screenshot below. New changes are too far indented:

image

Comment (AymenLoukil) [reply src/templates/base.html]: Thanks. I think it ok now :)

**Pull request merged**